ReactSkia/components/RSkComponentScrollView.cpp

```diff
@@ -291,6 +295,35 @@ ScrollStatus RSkComponentScrollView::scrollInDirection(RSkComponent* candidate,
   return scrollToFocus;
 }
 
+ScrollStatus RSkComponentScrollView::scrollTo(RSkComponent* candidate) {
+  if((candidate == nullptr) || (candidate == this)) {
+    return noScroll;
+  }
+
+  if(isVisible(candidate)) {
+    return noScroll;
+  }
+
+  RnsShell::ScrollLayer *scrollLayer = SCROLL_LAYER_HANDLE;
+  SkPoint currentScrollPos = scrollLayer->getScrollPosition();
+  SkIRect candidateFrame = candidate->getLayerAbsoluteFrame();
+  rnsKey direction;
+
+  //Check if need to scroll in backward/forward direction based on scroll position
+  if(isHorizontalScroll_) {
+     direction = RNS_KEY_Right;
+     if(candidateFrame.x() < currentScrollPos.x()) {
+       direction = RNS_KEY_Left;
+     }
+  } else {
+     direction = RNS_KEY_Down;
+     if(candidateFrame.y() < currentScrollPos.y()) {
+       direction = RNS_KEY_Up;
+     }
+  }
+  return handleScroll(direction,candidateFrame,false);
+}
+
```

Comment:
For the point 3: comment in "isVisible" API: is the comment to be "absFrame part of visible Frame" ? instead of intersect visible Frame

Reply:
yes right. updated the comment.